app/helpers/layout_helper.rb

```diff
@@ -2,4 +2,16 @@ module LayoutHelper
   def active_org(org)
     (org.id == current_organization.id ? 'active' : '')
   end
+
+  def colorize_transaction(transaction)
+    css_class = colorize_amount(transaction.amount)
+
+    if params[:controller] != 'transactions' && current_member.last_visited_at
+      if transaction.created_at > current_member.last_visited_at
+        css_class += ' new_transaction'
```

Comment:
@Gataniel how about using more deep color instead of opacity? I would suggest to use color that I see when I hover on Transaction line.